script.go

```diff
@@ -967,3 +968,36 @@ func newScanner(r io.Reader) *bufio.Scanner {
 	scanner.Buffer(make([]byte, 4096), math.MaxInt)
 	return scanner
 }
+
+// EncodeBase64 produces the base64 encoding of the input.
```

Comment:
Can we keep pipe methods all together, and preserve the existing alphabetical order? It just makes it easier to find stuff.